spec/renderer/integration/store/TimelineSpace/Contents/Mentions.spec.js

```diff
@@ -0,0 +1,152 @@
+  namespaced: true,
+  state: {
+    account: {
+      accessToken: 'token',
+      baseURL: 'http://localhost'
+    }
+  }
+}
+
+describe('Mentions', () => {
+  let store
+  let localVue
+
+  beforeEach(() => {
+    localVue = createLocalVue()
+    localVue.use(Vuex)
+    store = new Vuex.Store({
+      modules: {
+        Mentions: initStore(),
+        TimelineSpace: timelineState
+      }
+    })
+    Mastodon.mockClear()
+  })
+
+  describe('fetchMentions', () => {
+    it('should be updated', async () => {
+      const mockClient = {
+        get: () => {
+          return new Promise((resolve, reject) => {
+            resolve({
+              data: [
+                { id: 1, type: 'mention' },
+                { id: 2, type: 'favourite' },
+                { id: 3, type: 'reblog' },
+                { id: 4, type: 'follow' }
+              ]
+            })
+          })
+        }
+      }
+
+      Mastodon.mockImplementation(() => mockClient)
+      const mentions = await store.dispatch('Mentions/fetchMentions')
+      expect(store.state.Mentions.mentions).toEqual([
+        { id: 1, type: 'mention' },
+        { id: 2, type: 'favourite' },
+        { id: 3, type: 'reblog' },
+        { id: 4, type: 'follow' }
+      ])
+    })
+  })
+
+  describe('lazyFetchMentions', () => {
+    describe('last is null', ()  => {
+      it('should not be updated', async () => {
+        const result = await store.dispatch('Mentions/lazyFetchMentions', null)
+        expect(result).toEqual(null)
+      })
+    })
+
+    describe('loading', () => {
+      beforeAll(() => {
+        state = () => {
+          return {
+            lazyLoading: true,
+            heading: true,
+            mentions: [],
+            unreadMentions: [],
+            filter: ''
+          }
+        }
+      })
+      it('should not be updated', async () => {
+        const result = await store.dispatch('Mentions/lazyFetchMentions', {})
+        expect(result).toEqual(null)
+      })
+    })
+
+    describe('success', () => {
+      beforeAll(() => {
+        state = () => {
+          return {
+            lazyLoading: false,
+            heading: true,
+            mentions: [
+              { id: 1, type: 'mention' },
+              { id: 2, type: 'favourite' },
+              { id: 3, type: 'reblog' },
+              { id: 4, type: 'follow' }
+            ],
+            unreadMentions: [],
+            filter: ''
+          }
+        }
+      })
+      it('should be updated', async () => {
+        const mockClient = {
+          get: () => {
+            return new Promise((resolve, reject) => {
+              resolve({
+                data: [
+                  { id: 5, type: 'mention' },
+                  { id: 6, type: 'favourite' }
+                ]
+              })
+            })
+          }
+        }
+
+        Mastodon.mockImplementation(() => mockClient)
+        const mentions = await store.dispatch('Mentions/lazyFetchMentions', { id: 1 })
```

Comment:
__[ESLint]__ Error - 'mentions' is assigned a value but never used. (no-unused-vars) ([link](https://sider.review/gh/repos/124234495/pulls/849#issue-3855133))

<sub>posted by Sider</sub>